embc-app/Models/Db/HeadOfHousehold.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Gov.Jag.Embc.Public.Models.Db
 {
-    public abstract class EvacueeOld : Person
+    public abstract class EvacueeOld : Person, IAuditableEntity
```

Comment:
Is this required? This entity will go away, why do we need to change it?

Reply:
This was left over from the previous version, so it's not needed.  The will be dropped, once we move to TEST.

Reply:
Do we want to add the interface on this entity?